pages/common/fusermount.md

```diff
@@ -0,0 +1,16 @@
+# fusermount
+
+> Mount and unmount FUSE filesystems.
```

Comment:
I can´t find an example for mount using this command :thinking:
Maybe we should update the command description only for `unmount`.

Reply:
There is a `-o` option which expects mount options, so it should at least be possible to mount filesystems. But I couldn't find those options anywhere.

Reply:
Maybe those are the same options for `mount`, but that's just an asumption.

Reply:
The page doesn't need to have all possible examples, so we can leave it for someone else to pick up later.

Reply:
I think so.  It might even prompt a reader to say "why not?" and then fix it. 🤷🏻